device.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/01org/ciao/ssntp/uuid"
```

Comment:
@amshinde @jodh-intel @sameo I think its time we moved the uuid implementation to our own utils. That way once ciao-qemu is made its own repo we do not pull in rest of ciao.

Reply:
@mcastelino Yes, agreed.

Reply:
@mcastelino Yup, I will move over the uuid implementation.